walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
I didn't see this function.
Was it added in an earlier round of commits?

Reply:
It's a part of the walkthrough tutorials example code! It's not in my actual portfolio code.

Reply:
Ah, I see. 
I didn't look closely at the file path.